frontend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "3",
+    "prop-types": "^15.6.1",
+    "react": "^16.3.2",
+    "react-bootstrap": "^0.32.1",
+    "react-dom": "^16.3.2",
+    "react-helmet": "^5.2.0",
+    "react-scripts": "1.1.4",
+    "react-table": "^6.8.2"
+  },
+  "scripts": {
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test --env=jsdom",
+    "eject": "react-scripts eject",
+    "lint": "eslint 'src/**/*.js'",
```

Comment:
in `cat` we also run `prettier` when linting
```yml
    "lint": "eslint 'src/**/*.js' && prettier -l 'src/**/*.js'",
```

Reply:
If I'm not mistaken this is not needed when we use `eslint-plugin-prettier`.